packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import React, { ReactElement, useMemo } from 'react';
+import { useRouter } from 'next/router';
+import { Tab, TabContainer } from '../tabs/TabContainer';
+import NotificationsBell from '../notifications/NotificationsBell';
+
+enum FeedNavTab {
+  ForYou = 'For you',
+  Popular = 'Popular',
+  Bookmarks = 'Bookmarks',
+  History = 'History',
+}
+
+function FeedNav(): ReactElement {
+  const router = useRouter();
+
+  const shouldRenderNav = useMemo(() => {
+    return /(?<!\/.+)\/(?:my-feed|popular|upvoted|discussed|bookmarks|history|notifications|)$/.test(
```

Comment:
We converted to .includes on the other one where evaluation was like this, do we want to do it here as well?

Reply:
includes here might not fit the requirement, as we don't want to match paths that are nested, eg: `/{username}/upvoted` should not return true here. We can for sure do it without regex as well, but not sure if it's with it.

Reply:
No worries, didn't pay much attention to it just saw the pattern, so it's fine anyways.

Reply:
Updated to check against `feedName` as opposed to matching the route path 🙏